.github/workflows/cd.yml

```diff
@@ -89,9 +84,14 @@ jobs:
           git config --global user.email 'yiz@microsoft.com'
 
       - name: release alpha npm packages to npmjs.org
-        if: ${{ github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/prerelease' }}
+        if: ${{ github.ref == 'refs/heads/dev' }}
         run: |
-          npx lerna version prerelease --preid=alpha.$(git rev-parse --short HEAD) --exact --no-push --allow-branch ${GITHUB_REF#refs/*/} --yes
+          npx lerna version prerelease --preid=alpha.$(git rev-parse --short HEAD) --exact --no-push --allow-branch dev --yes
+
+      - name: release beta packages to npmjs.org
+        if: ${{ github.ref == 'refs/heads/prerelease' }}
+        run: |
+          npx lerna version prerelease --preid=beta.$(date "+%Y%m%d%H") --exact --no-push --allow-branch prerelease --yes
```

Comment:
I'm afraid if two beta would be released in one hour so that the preid would have conflicts then.

Reply:
even release twice in one hour, still success on pipeline